service/awsclusterconfig/v1/resource/awsconfig/resource.go

```diff
@@ -0,0 +1,61 @@
+package awsconfig
+
+import (
+	"github.com/giantswarm/apiextensions/pkg/apis/provider/v1alpha1"
+	"github.com/giantswarm/microerror"
+	"github.com/giantswarm/micrologger"
+	"k8s.io/client-go/kubernetes"
+)
+
+const (
+	// Name is the identifier of the resource.
+	Name = "awsconfigv10"
```

Comment:
That is not right.

Reply:
What would be right? I thought this refers to corresponding resource and I looked that v10 is currently the latest `AWSConfig` version.

Reply:
The version here refers to the resource version it is in. So it should be v1. This version here is not reflecting internals of another operator. Their lifecycles are quite different and we will not keep versions in sync across projects. They are all independent in that sense.

Reply:
Ok. Will also fix the corresponding `kvmconfig` here as well.